test/acceptance/tests/basic_test.go

```diff
@@ -31,8 +31,9 @@ type SetupConfig struct {
 
 func TestBasic(t *testing.T) {
 	cases := map[string]struct {
-		secure     bool
-		enterprise bool
+		secure                 bool
+		enterprise             bool
+		autoPublishRegistrator bool
```

Comment:
Given the current test setup this only tests the case where the private ecr repo has already been created. We need to test the case where our module creates the ecr repo before the docker push.

Reply:
addressed in the previous comment